aikit/ml_machine/ml_machine_launcher.py

```diff
@@ -97,12 +102,14 @@ def _process_command_arguments(self, parser=None):
 
         args = parser.parse_args()
 
-        if args.command is None:
-            raise ValueError("Unknown command please choose among 'start','stop','controller', 'worker' or 'result'")
+        allowed_commands = [c.value for c in self.Commands]
+        if args.command is None or args.command.lower() not in allowed_commands:
+            raise ValueError("Unknown command please choose among %s" % ', '.join(allowed_commands))
 
```

Comment:
So that the error message cleary indicates what are the possible command.
Before it was failing when created the Command (enum) and it didn't say what were the possible values